fidesops.toml

```diff
@@ -25,3 +25,4 @@ TASK_RETRY_COUNT=2
 TASK_RETRY_DELAY=5
 TASK_RETRY_BACKOFF=2
 REQUIRE_MANUAL_REQUEST_APPROVAL=false
+MASKING_STRICT=true
```

Comment:
By default, `updates` only are used in our internal fidesops.toml, but you can set to `false` within an integration test to test gdpr delete/delete.